### PR TITLE
feat: kiva credit stats for portfolio overview page MARS-352

### PR DESCRIPTION
**Pull request by emuvente**
Credit summary lightbox is still in-progress and will be completed in another ticket.

## Mobile
Screen Shot 2023-03-16 at 4 29 12 PM

## Desktop
Screen Shot 2023-03-16 at 4 28 11 PM

### CODE REVIEW

File: src/pages/Portfolio/CreditSummaryLightbox.vue [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+<script>
+export default {
+	name: 'CreditSummaryLightbox',
+	data() {
+		return {
+			value: 1234.56,
+		};
+	},
+	computed: {
+		// depositSubTotal() {
```
Comment (dyersituations) [inline src/pages/Portfolio/CreditSummaryLightbox.vue:60]: Future work?

**Review by mcstover — APPROVED** [review]
Step by Step!

**Review by christian14b — APPROVED** [review]
Cool!

File: src/pages/Portfolio/CreditSummaryLightbox.vue (hunk 1 above)
Comment (emuvente) [reply src/pages/Portfolio/CreditSummaryLightbox.vue:60]: Yes, this whole lightbox is future work, MARS-361.

**Review by dyersituations — APPROVED** [review]

**Pull request merged**